### DISCUSSION TITLE
Using local git repository

### DISCUSSION THREAD

**Discussion by ericsonj** [gomods/athens#1738] [General] [answered]
Hi, is possible configure to use local GIT directory for get the module ?

E.g:

go get example.com/my/module <--- PROXY <--- file:///repositories/module (es a git directory)

**Comment by matt0x6F** [comment, marked as answer]
Are you referring to the proxy indicating that Go should pull a file from the same machine that runs `go get`? If so, that's what `replace` is for.

If you're looking to prepopulate Athens with files, you can [pre-fill disk cache](https://docs.gomods.io/configuration/prefill-disk-cache/).
